### PR TITLE
feature: dm-280

### PR DESCRIPTION
**Pull request by itsmostafa**
Add check categories command to Palo Alto

## 💭 Description

Palo Alto is missing a check category command to check on domain categories

## ✅ Checklist

- [X] My code follows the code style of this project.
- [X] My changes have been adequately tested locally.
- [X] All automated tests are passing.
- [X] I have updated/added required automated tests.
- [ ] Pre-commit checks are passing.

### CODE REVIEW

File: src/utils/proxies/__init__.py [hunk 1]
```diff
@@ -62,15 +62,15 @@
...
-        "check_url": "https://talosintelligence.com/reputation_center",
+        "check_url": None,
         "check_category_func": None,
         "categorize_func": None,
     },
     {
         "name": "IBM X Force",
         "url": "https://exchange.xforce.ibmcloud.com/",
         "categorize_url": "https://exchange.xforce.ibmcloud.com/",
-        "check_url": "https://exchange.xforce.ibmcloud.com/",
```
Comment (zenine07) [inline src/utils/proxies/__init__.py]: Why did you remove the check_urls in these two locations? The check url is only used for the frontend so that somebody can go manually check it.

Comment (itsmostafa) [reply src/utils/proxies/__init__.py]: I didn't know it was used for that. It was failing in the proxy test harness because its checking if that variable exists before it runs the category check.

Comment (zenine07) [reply src/utils/proxies/__init__.py]: Ahh, yes, it would fail as I'm looking at it. Let me check how I fixed it when we actually process it.

Comment (zenine07) [reply src/utils/proxies/__init__.py]: Actually should be good if it's added back in. It looks like the `process` function will fail, but on exception it returns None instead of the category. Which is what should happen.

**Review by zenine07 — APPROVED** [review]
